xml/Microsoft.Extensions.Primitives/ChangeToken.xml

```diff
@@ -90,7 +90,7 @@
         <Parameter Name="state" Type="TState" />
       </Parameters>
       <Docs>
-        <typeparam name="TState">To be added.</typeparam>
+        <typeparam name="TState">The type of the state to register the consumer action for.</typeparam>
```

Comment:
This typeparam description is a bit difficult to understand as it is. How about:
```suggestion
        <typeparam name="TState">The type of the state for the <paramref name="changeTokenConsumer" /> action.</typeparam>
```